google/cloud/aiplatform/training_jobs.py

```diff
@@ -4000,15 +4714,6 @@ def __init__(
             model_encryption_spec_key_name=model_encryption_spec_key_name,
         )
 
-        self._column_transformations = (
-            column_transformations_utils.validate_and_get_column_transformations(
-                column_specs, column_transformations
-            )
-        )
-
-        self._optimization_objective = optimization_objective
-        self._additional_experiments = []
-
     def run(
```

Comment:
Can you just remove the `run` override and defer to super?

Reply:
If we remove this, Sphinx will not generate documentation for the `run()` function. It can pull the docstring from the parent class (tested with `nox -s docs`), but I couldn't find a way to make it document the function signature other than by writing it out manually.

Reply:
Hm, that's quite annoying. Let me ask the SDK team for more solutions.